src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskExecuteReconfigureControllerTest.java

```diff
@@ -161,23 +226,33 @@ public void should_return_a_204_after_tasks_are_marked_and_executed_for_reconfig
 
         result.then().assertThat()
             .statusCode(HttpStatus.OK.value())
-            .body("tasks.size()", lessThanOrEqualTo(10)) //Default max results
-            .body("tasks.id", everyItem(notNullValue()))
-            .body("tasks.id", hasItem(is(taskId)));
+            .body("tasks.size()", equalTo(0)); //Default max results
+
+        // execute reconfigure process is not performed on current task
+        // retry window is set 0 hours, so 1 unprocessed reconfiguration record to report
+        sleep(10000);
+        result = restApiActions.post(
+            ENDPOINT_BEING_TESTED,
+            taskOperationRequestForExecuteReconfiguration(
+                TaskOperationType.EXECUTE_RECONFIGURE_FAILURES,
+                OffsetDateTime.now().minus(Duration.ofDays(1))
+            ),
+            assigneeCredentials.getHeaders()
```

Comment:
should include the unprocessed record in an assertion?

Test name also says "find unprocessed log message", but we do not verify the log message is set as expected due to the failure